salt/modules/win_servermanager.py

```diff
@@ -16,24 +16,43 @@
 
 log = logging.getLogger(__name__)
 
+__virtualname__ = 'win_servermanager'
+
 
 def __virtual__():
     '''
-    Load only on windows
+    Load only on windows with servermanager module
     '''
-    if salt.utils.is_windows():
-        return 'win_servermanager'
-    return False
+    if not salt.utils.is_windows():
+        return (False, 'Failed to load win_servermanager module:\n'
+                       'Only available on Windows systems.')
+
+    if not _check_server_manager():
+        return (False, 'Failed to load win_servermanager module:\n'
+                       'ServerManager module not available.\n'
+                       'May need to install Remote Server Administration Tools.')
+
+    return __virtualname__
 
 
-def _srvmgr(func):
+def _check_server_manager():
     '''
-    Execute a function from the ServerManager PS module and return the STDOUT
+    See if ServerManager module will import
+
+    Returns: True if import is successful, otherwise returns False
     '''
-    return __salt__['cmd.run'](
-            'Import-Module ServerManager ; {0}'.format(func),
-            shell='powershell',
-            python_shell=True)
+    return not __salt__['cmd.retcode']('Import-Module ServerManager',
+                                       shell='powershell',
+                                       python_shell=True)
```

Comment:
@cachedout The powershell command requires python_shell=True. The functions have been using cmd_quote on passed parameters for this reason.